tests/widget/modal/widget-modal.email-suspend.test.js

```diff
@@ -13,7 +13,29 @@ test('Modal widget with email suspend', async ({ page }) => {
   await page.getByLabel('Username').fill('demouser');
   await page.getByLabel('Password').fill('j56eKtae*1');
 
-  await clickButton('Next', '/authenticate');
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.route('*/**/authenticate/**', async (route) => {
```

Comment:
Unless I'm mistaken, I think it might be best to declare this `page.route` before the button click. That way, you guarantee that the resulting request gets intercepted.

Reply:
Moved this